NPE within the JDTTreeBuilder#createExecutableReferenceExpression method

I used spoon to build the model of a bigger project using java 8 features. Unfortunately I got an NPE within the method createExecutableReferenceExpression from the JDTTreeBuilder class. 

Here is the exact stacktrace: 

```
Caused by: java.lang.NullPointerException
	at spoon.support.compiler.jdt.JDTTreeBuilder.createExecutableReferenceExpression(JDTTreeBuilder.java:1505)
	at spoon.support.compiler.jdt.JDTTreeBuilder.visit(JDTTreeBuilder.java:1495)
	at org.eclipse.jdt.internal.compiler.ast.ReferenceExpression.traverse(ReferenceExpression.java:755)
	at spoon.support.compiler.jdt.JDTTreeBuilder.visit(JDTTreeBuilder.java:2192)
	at org.eclipse.jdt.internal.compiler.ast.MessageSend.traverse(MessageSend.java:1005)
	at spoon.support.compiler.jdt.JDTTreeBuilder.visit(JDTTreeBuilder.java:2188)
	at org.eclipse.jdt.internal.compiler.ast.MessageSend.traverse(MessageSend.java:1005)
	at org.eclipse.jdt.internal.compiler.ast.ReturnStatement.traverse(ReturnStatement.java:373)
	at spoon.support.compiler.jdt.JDTTreeBuilder.visit(JDTTreeBuilder.java:2272)
	at org.eclipse.jdt.internal.compiler.ast.MethodDeclaration.traverse(MethodDeclaration.java:317)
	at spoon.support.compiler.jdt.JDTTreeBuilder.visit(JDTTreeBuilder.java:2946)
	at org.eclipse.jdt.internal.compiler.ast.TypeDeclaration.traverse(TypeDeclaration.java:1338)
	at org.eclipse.jdt.internal.compiler.ast.CompilationUnitDeclaration.traverse(CompilationUnitDeclaration.java:745)
	at org.eclipse.jdt.internal.compiler.ast.CompilationUnitDeclaration.traverse(CompilationUnitDeclaration.java:706)
	at spoon.support.compiler.jdt.JDTBasedSpoonCompiler.buildSources(JDTBasedSpoonCompiler.java:229)
	at spoon.support.compiler.jdt.JDTBasedSpoonCompiler.build(JDTBasedSpoonCompiler.java:505)
	at org.cubeengine.pericopist.extractor.java.JavaMessageExtractor.extract(JavaMessageExtractor.java:93)
```

I can't say which line of the project causes this exception. 

But I know it from the Spoon side. The ResourceBuilder#getExecutableReference returns null if the referenceExpression.binding value is null. This case isn't handled here.

--- a/pom.xml
+++ b/pom.xml
@@ -186,7 +186,7 @@
     <dependency>
       <groupId>org.eclipse.jdt.core.compiler</groupId>
       <artifactId>ecj</artifactId>
-      <version>4.4</version>
+      <version>4.4.2</version>
     </dependency>
     <dependency>
       <groupId>junit</groupId>

